data.js

```diff
@@ -5657,6 +5657,14 @@ module.exports = [{
     url: "https://github.com/StephanHoyer/classies",
     source: "https://github.com/StephanHoyer/classies/blob/master/index.js"
   },
+  {
+    name: "Spck-Embed",
+    github: "spckio/spck-embed",
+    tags: ["embeddable", "code", "editor"],
+    description: "Small embeddable online code editor for demos.",
+    url: "https://github.com/spckio/spck-embed",
+    source: "https://raw.githubusercontent.com/spckio/spck-embed/master/dist/spck-embed.min.js"
```

Comment:
Please change the `source link to `https://raw.githubusercontent.com/spckio/spck-embed/master/dist/spck-embed.js`. The readme mentions that source links should be to the unminified source code.

Reply:
I have changed the source link. Thanks.